beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -167,14 +172,125 @@ def normalize(tensor):
 
 print("Median relative difference between original and MuLaw reconstucted signals: {:.2%}".format(err))
 
+######################################################################
+# ``torchaudio`` also supports various ``augmentations``:
```

Comment:
There is no need to make the distinction between transforms and augmentations. You can just add those to the list of transforms.